packages/rocketchat-integrations/server/api/api.js

```diff
@@ -236,9 +240,19 @@ function executeIntegrationRest() {
 			sandbox.script = script;
 			sandbox.request = request;
 
-			const result = vm.runInNewContext('script.process_incoming_request({ request: request })', sandbox, {
+			const result = Future.fromPromise(vm.runInNewContext(`
+				new Promise((resolve, reject) => {
```

Comment:
please, add a `catch` to this `Promise`, otherwise calling `reject` will thrown a `unhandled promise rejection`